compiler/damlc/daml-doc/test/DA/Daml/GHC/Damldoc/Tests.hs

```diff
@@ -259,11 +262,10 @@ damldocExpect importPathM testname input check =
 
     case mbResult of
       Left err -> assertFailure $ unlines
-                  ["Parse error(s) for test file " <> testname, show err]
-
-      -- first module is the root we started from, so is the one that must satisfy the invariants
-      Right ms -> check $ head ms
+                  ["Parse error(s) for test file " <> testfile, show err]
```

Comment:
Could you fix the indentation here? This is a bit much.

Reply:
I inherited that but I'm happy to change it.

Reply:
Thank you :-)